dom/nodes/Node-cloneNode.html

```diff
@@ -34,6 +34,13 @@
 }
 
 function check_copy(orig, copy, type) {
+    assert_not_equals(orig, copy, "Object equality");
+    assert_equal_node(orig, copy, "Node equality");
+    assert_true(orig.constructor.name === type.name, "original instanceof " + type);
+    assert_true(copy.constructor.name === type.name, "copy instanceof " + type);
```

Comment:
If you use assert_equals here you'll get more helpful error messages and the message argument can basically be "original" and "copy", respectively.